docs/tutorials-samples/targeting-core-for-new-libraries.md

```diff
@@ -0,0 +1,298 @@
+# Targeting with .NET Core Libraries
+
+Targeting .NET Core for new libraries can be done a number of ways, depending on the platform(s) you wish to support.
+
+## How do I target .NET Core?
+
+First things first:
+
+For the purposes of building a library, targeting ".NET Core" means targetting the .NET Standard Platform.  [This document](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md) outlines everything you need to know, in-depth, about what that means.
```

Comment:
When you say this document, it seems to imply you're talking about the current topic and not a different one. Saying that the link covers everything you need to know, in-depth, seems a bit too much. Maybe you should say "To see what that means exactly, see [.NET Platform Standard](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md)." Also putting a friendlier link name helps SEO.